components/FlexibleHeader/src/MDCFlexibleHeaderViewController.h

```diff
@@ -74,6 +74,34 @@
  */
 @property(nonatomic, weak, nullable) UIViewController *topLayoutGuideViewController;
 
+/**
+ Whether the view controller should attempt to extract safe area insets from the view controller
+ hierarchy or not.
+
+ When this property is enabled, the flexible header will infer the top safe area inset for the
+ flexible header view based on the header view controller's root ancestor view controller.
+
+ When this property is disabled, the flexible header will infer the top safe area inset using the
+ device's inferred top safe area insets. This assumes that the flexible header consumes the entire
+ screen. If this is not the case, such as in a popover or an iPad modal sheet, consider enabling
+ this property.
+
+ This behavior will eventually be enabled by default.
+
+ Default is NO.
```

Comment:
Is there no way to determine this from view.self screen coordinates and the windows safe area inset?
The answer may be no, but this seem like a significant piece of complexity I'd like to avoid if possible.

Reply:
Discussed offline, but unfortunately this is not possible because the frame of our flexible header view can shift on and off-screen, affecting our ability to read a consistent safe area inset value.